### ISSUE TITLE
Setting a custom line width is no longer working

### ISSUE THREAD

**Issue by ferrolho** [rdeits/MeshCat.jl#246]
While sanitising the list of open issues, I noticed that setting a custom line width is no longer working.

Here is a simple snippet for displaying a line through a set of random points:

```julia
using Colors
using MeshCat

vis = Visualizer()
open(vis)

points = [rand(3) for _ = 1:100]
pointcloud = PointCloud(points)

material = LineBasicMaterial(color=colorant"yellow", linewidth=8)
setobject!(vis, Object(pointcloud, material, "Line"))
```

which results in the following:

<img width="1015" alt="image" src="https://github.com/rdeits/MeshCat.jl/assets/4166093/0bd1826e-802f-4089-be59-325a3b2e9f13">

Notice how the width of the rendered line is `1` despite setting `linewidth=8`.

**Comment by ferrolho** [comment]
Hmm, apparently this is not going to be fixed for GL — see https://bugs.chromium.org/p/chromium/issues/detail?id=675308.

But there is an example with another way of achieving line thickness: https://threejs.org/examples/#webgl_lines_fat

**Comment by ferrolho** [comment]
I've included and exported `Line2`, `LineMaterial`, and `LineGeometry` in https://github.com/ferrolho/meshcat/commit/8fdb46b5ec4cd84d266ebe29b277a1fe7e8ad32c, but I am not sure if that is the proper way to do it. And to be honest, I have no idea what to do next on the MeshCat.jl end... For example, I suppose we need to define a Julia struct for `LineGeometry` in `src/geometry.jl`, right?

**Comment by soulslicer** [comment]
Any update on this?

**Comment by ferrolho** [comment]
> Any update on this?

Yes! I have added support for Line2, LineMaterial, and LineGeometry.

There are two open PRs that will enable this: https://github.com/meshcat-dev/meshcat/pull/193 and https://github.com/rdeits/MeshCat.jl/pull/270.

Here is a snippet equivalent to the one on the original post demonstrating the new functionality:

```julia
using Colors
using MeshCat
using GeometryBasics

vis = Visualizer()
open(vis)

points = [Point(rand(3)...) for _ = 1:100]

material = FatLineMaterial(color=colorant"yellow", linewidth=4)
setobject!(vis, FatLine(points, material))
```

(using line width set to 4 instead of 8 because that was too thick...)

which results in the following:

<img width="955" height="535" alt="Image" src="https://github.com/user-attachments/assets/093b8496-aa95-4577-93b6-1908ead747db" />
